ParseAligns/abyss-fixmate.cc

```diff
@@ -452,14 +459,14 @@ int main(int argc, char* const* argv)
 		+ numFR + numRF + stats.numFF
 		+ stats.numDifferent;
 	cerr <<
-		"Mateless   " << percent(alignments.size(), sum) << "\n"
-		"Unaligned  " << percent(stats.bothUnaligned, sum) << "\n"
-		"Singleton  " << percent(stats.oneUnaligned, sum) << "\n"
-		"FR         " << percent(numFR, sum) << "\n"
-		"RF         " << percent(numRF, sum) << "\n"
-		"FF         " << percent(stats.numFF, sum) << "\n"
-		"Different  " << percent(stats.numDifferent, sum) << "\n"
-		"Total      " << sum << endl;
+		"Mateless (read has no mate or input ordering is wrong) " << percent(alignments.size(), sum) << "\n"
+		"Unaligned (both reads unaligned)                       " << percent(stats.bothUnaligned, sum) << "\n"
+		"Singleton (one read aligned, one unaligned)            " << percent(stats.oneUnaligned, sum) << "\n"
+		"FR (forward-reverse, commonly paired-end reads)        " << percent(numFR, sum) << "\n"
+		"RF (reverse-forward, true mate-pair reads)             " << percent(numRF, sum) << "\n"
+		"FF (unsupported by abyss)                              " << percent(stats.numFF, sum) << "\n"
+		"Different (each read maps to a different contig)       " << percent(stats.numDifferent, sum) << "\n"
+		"Total                                                  " << sum << endl;
```

Comment:
I'd prefer to move this information to documentation. The Wiki is publicly editable. Feel free to create a new page at https://github.com/bcgsc/abyss/wiki

Reply:
Umm, that was the main reason I wanted to clarify the *log file*. People want to copy&paste relevant info from the logs and the columns should be self-explanatory. You are free to reject me. I do not say including this text in addition into the wiki pages too would be that much work but I haven't heard of those pages before. Abyss has disparate documentation scattered in runtime help texts, all the README.md files, I would rather improve clarify of the log which are the primary source of the information.

Reply:
There's a limit to how much documentation you can include in the output of the program itself. I'd prefer to keep the output terse and improve the documentation, which there's more room for explanation. I'd like each tool of ABySS to have a page of documentation, like Sealer does, to describe its output.